### DISCUSSION TITLE
Added DallE Prompt plugin.

### DISCUSSION THREAD

**Discussion by TheCompAce** [TheCompAce/Auto-GPT-Powershell#9] [Show and tell]
This is the first version of the "DallE Prompt" plugin. It sets a System message to create <DALLE dest="filename">value</DALLE> tags, that has the prompt, then it takes that tag and downloads it to the session folder, in a "TTI" folder, and in there you have a "prompts.json" of the file and prompt used to make it. It also adds to that folder a "gallery.htm" file that has "prompts.json' embodied to show all the images from one place.

I have plans on improving then, and adding Stable Diffusion Web-UI API support.

**Comment by TheCompAce** [comment]
I have also added "Stable Diffusion" so you can use this offline.

**Comment by Kallamamran** [comment]
Great, but I still have problems running it at all... Either I'm to stupid to understand the readme or the readme is not entirely complete ;)
